docs/data-cookbooks/browse-generation.md

```diff
@@ -78,173 +76,205 @@ For this example, you are going to be adding two workflows to your Cumulus deplo
 
 #### Workflow Configuration
 
-Add the following to a new file ```browseExample.yml``` in your deployment's main directory (the same location your app directory, lambdas.yml, etc are), copy the example file [from github](https://github.com/nasa/cumulus/blob/master/example/workflows/browseExample.yml).  The file should contain the two example workflows.
+Add the following to a new file ```browse_example.tf``` in your deployment's main directory: [from github](https://github.com/nasa/cumulus/blob/master/example/workflows/browseExample.yml).  The file should contain the two example workflow modules.
```

Comment:
```suggestion
Add the following to a new file `browse_example.tf` in your deployment's main directory: [from github](https://github.com/nasa/cumulus/blob/master/example/workflows/browseExample.yml). The file should contain the two example workflow modules.
```

Nitpick: Unless there a reason for the triple backticks vs single backticks, I would kind of prefer to standardize on the latter. Also the double spaces after the period.